src/plugin/oauth/index.js

```diff
@@ -2166,32 +2167,48 @@ class BaseOauthPlugin extends BasePlugin {
       ) {
         await plugin.set_introspection_cache(session_id, response);
       }
+      plugin.server.logger.verbose("Assertions: PASSED introspect_access_token")
+    } else {
+      plugin.server.logger.verbose("Assertions: SKIPPED introspect_access_token")
     }
 
-    if (
-      pluginStrategy == PLUGIN_STRATEGY_OIDC &&
```

Comment:
This check needs to remain.

Reply:
I don't understand this, is id_token assertions not available in OAuth2?